### PR TITLE
Add Russian readme

### PR DESCRIPTION
**Pull request by XORandom**
The Ru-version of the README, and links to ru.rst have also been added.

### CODE REVIEW

**Review by MyreMylar — APPROVED** [review]
OK, LGTM 👍 

Reverse translation via AI looks fine, other than that I will have to take it on trust as I'm not a native reader of Russian. 

If anyone else in the community is, please add a review!

File: docs/readmes/README.ja.rst [hunk 1]
```diff
@@ -232,9 +233,12 @@ gfxdraw には SDL_gfx の埋め込みバージョンが含まれています。
 .. _Pygame Community Discord Server: https://discord.gg/pygame
 .. _wiki pages: https://github.com/pygame-community/pygame-ce/wiki
 
-.. _简体中文: ./docs/readmes/README.zh-cn.rst
+.. _English: ./../../README.rst
+.. _简体中文: README.zh-cn.rst
 .. _繁體中文: README.zh-tw.rst
-.. _Français: ./docs/readmes/README.fr.rst
-.. _فارسی: ./docs/readmes/README.fa.rst
-.. _Español: ./docs/readmes/README.es.rst
```
Comment (XORandom) [inline docs/readmes/README.ja.rst:239]: "../docs/readmes" paths would not work in this file

**Review by XORandom — COMMENTED** [review]
need to remove unnecessary newlines

**Review by ankith26 — COMMENTED** [review]
You missed the modifications needed for the newly added italian and zh-tw READMEs

**Comment by XORandom** [comment]
Added the necessary links to the Italian README

File: docs/readmes/README.ja.rst (hunk 1 above)
Comment (ankith26) [reply docs/readmes/README.ja.rst:239]: Nice fix!

**Review by ankith26 — APPROVED** [review]
Thanks for contributing to pygame-ce! :partying_face: 

Approving this PR after a quick sanity check with google translate.

**Comment by ankith26** [comment]
Just added a minor commit fixing formatting to get this PR in

**Pull request merged**